resolver/contentResolver.go

```diff
@@ -0,0 +1,70 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+)
+
+var serviceNameRegex = regexp.MustCompile("__(.*?)/")
+
+type contentResolver struct {
+	contentResolverAppURI  string
+	contentResolverAppName string
```

Comment:
I actually didn't, I made a constructor NewContentResolver(contentResolverAppURI string) and it's initialised in it from the URI. Anyways, if you have a better suggestion, please let's talk.